docs/en/observability/logs-troubleshooting.asciidoc

```diff
@@ -1,4 +1,133 @@
+to the role of the authenticated user.
+----
+
+[discrete]
+[[logs-troubleshooting-insufficient-priv-solution]]
+=== Solution
+
+You need to either:
+
+* Have an administrator give you the `monitor` and `manage_own_api_key` cluster privileges and the `auto_configure` and `create_doc` indices privileges. Once you have these privileges, restart the onboarding flow.
+* Get an API key from an administrator and manually add the API to the {agent} configuration. See <<logs-stream-agent-config, Configure the {agent}>> for more on manually updating the configuration and adding the API key.
+
+[discrete]
+[[logs-troubleshooting-API-key-failed]]
+== Failed to create API key
+
+If you don't have the privileges to create `savedObjects` in {kib}, you'll see the following error message:
+
+[source, plaintext]
+----
+Failed to create API key
+
+Something went wrong: Unable to create observability-onboarding-state
+----
+
+[discrete]
+[[logs-troubleshooting-API-key-failed-solution]]
+=== Solution
+
+You need an administrator to give you the `Saved Objects Management` {kib} privilege to generate the required `observability-onboarding-state` flow state.
+Once you have the necessary privileges, restart the onboarding flow.
+
+[discrete]
+[[logs-troubleshooting-kib-not-accessible]]
+== {kib} not accessible from host
+
+If {kib} is not accessible from the host, you'll see the following error message after pasting the *Install the {agent}* instructions into the host:
+
+[source, plaintext]
+----
+Failed to connect to {host} port {port} after 0 ms: Connection refused
+----
+
+[discrete]
+[[logs-troubleshooting-kib-not-accessible-solution]]
+=== Solution
+
+The host needs access to {Kib}. If that's not possible, you can manually download and configure the {agent}. See the <<logs-stream, Stream a log file>> documentation for more information.
+
+[discrete]
+[[logs-troubleshooting-download-agent]]
+== Download {agent} failed
+
+If the host was able to download the installation script but cannot connect to the public artifact repository, you'll see the following error message:
+
+[source, plaintext]
+----
+Download Elastic Agent
+
+Failed to download Elastic Agent, see script for error.
+----
+
+[discrete]
+[[logs-troubleshooting-download-agent-solution]]
+=== Solutions
+
+* If the combination of the {agent} version and operating system architecture is not available, you'll see the following error message:
++
+[source, plaintext]
+----
+The requested URL returned error: 404
+----
++
+To fix this, update the {agent} version in the installation instructions to a known version of the {agent}. 
+* If the {agent} was fully downloaded previously, you'll see the following error message:
++
+[source, plaintext]
+----
+Error: cannot perform installation as Elastic Agent is already running from this directory
+----
++
+To fix this, delete previous downloads and restart the onboarding.
+* You're an Elastic Cloud Enterprise user without access to the Elastic downloads page.
```

Comment:
Not sure if there is a solution to this one that we can add as it's more of a cause at this point.